e2e/wdio/core/tests/slider.e2e-spec.ts

```diff
@@ -76,14 +76,16 @@ describe('slider test suite', function() {
         });
     });
 
-    xdescribe('tick marks and labels examples', function() {
+    describe('tick marks and labels examples', function() {
         it('should check tick marks', () => {
```

Comment:
if you comment out the assertion, no need to leave scrollintoview, just skip the test case `xit`